README.md

```diff
@@ -91,36 +95,68 @@ It will look like the following:
 Run the following if you want to give someone else access to completely deployed and existing environment.
 
 1. Install the [Azure CLI](https://learn.microsoft.com/cli/azure/install-azure-cli)
-1. Run `azd init -t azure-search-openai-demo`
-1. Run `azd env refresh -e {environment name}` - Note that they will need the azd environment name, subscription Id, and location to run this command - you can find those values in your `./azure/{env name}/.env` file.  This will populate their azd environment's .env file with all the settings needed to run the app locally.
-1. Run `pwsh ./scripts/roles.ps1` - This will assign all of the necessary roles to the user so they can run the app locally.  If they do not have the necessary permission to create roles in the subscription, then you may need to run this script for them. Just be sure to set the `AZURE_PRINCIPAL_ID` environment variable in the azd .env file or in the active shell to their Azure Id, which they can get with `az account show`.
+2. Run `azd init -t azure-search-openai-demo`
+3. Run `azd env refresh -e {environment name}` - Note that they will need the azd environment name, subscription Id, and location to run this command - you can find those values in your `./azure/{env name}/.env` file.  This will populate their azd environment's .env file with all the settings needed to run the app locally.
+4. Run `pwsh ./scripts/roles.ps1` - This will assign all of the necessary roles to the user so they can run the app locally.  If they do not have the necessary permission to create roles in the subscription, then you may need to run this script for them. Just be sure to set the `AZURE_PRINCIPAL_ID` environment variable in the azd .env file or in the active shell to their Azure Id, which they can get with `az account show`.
+
+### Switching from an Azure OpenAI endpoint to an OpenAI instance
+
+If you want to use an existing OpenAI instance, before running `azd up` to create your Azure resources and deploy the app on Azure, follow these steps:
+
+1. Navigate to the [main.bicep](./infra/main.bicep) file in the infra folder
+2. Look for the `openAiType` parameter at line 16
+3. Change its value to `openai` (instead of `azure`)
```

Comment:
AZD has parameters. We shouldn't require the user to manually change the bicep file. Alternatively, use the `azd set` command with environment variables

Reply:
Agreed. We may want to consider splitting it into a separate bicep module as well.